docs/pages/toolpad/index.js

```diff
@@ -14,6 +15,11 @@ import StudioIntro from '../../src/components/landing/StudioIntro';
 export default function Home() {
   return (
     <BrandingCssVarsProvider>
+      <Head
+        title="Toolpad Core: Open-source dashboard framework for React"
+        description="Toolpad Core offers the components needed for your next admin panel and internal tools project."
```

Comment:
@prakhargupta1 This title/description was missing, might need better copy. Also may want to update the twitter card image